frontend/src/components/table/TableQuickInput.js

```diff
@@ -133,6 +185,17 @@ class TableQuickInput extends Component {
     }
   }
 
+  setRef = (ref) => {
+    this.form = ref;
+  };
+
+  setWidgetsRef = (c) => {
```

Comment:
pls use a diff name with more meaning for the name. Something like `componentNode` or `nodeRef` instead of just `c`